### PR TITLE
Create ukrainian_spt

### PR DESCRIPTION
**Pull request by oleksaboiko**
Hello!
Here's our case study, which we sent you a couple of days ago)

Below are the images needed:

[images/ukraine_spt1.png](https://github.com/oleksaboiko/first/blob/683bd4095c8210e51a60684fcf9758a211828d5c/images/ukraine_spt1.png)

[images/ukraine_spt2.png](https://github.com/oleksaboiko/first/blob/683bd4095c8210e51a60684fcf9758a211828d5c/images/ukraine_spt2.png)

### CODE REVIEW

File: source/site/about/case_studies/ukrainian_spt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The initiative group was coordinated by its organizers, with regular meetings where tasks, proposals, questions, and problems that arose during the work were discussed.
+Most of the communication was in the messenger chat, where constructive discussions were held between the participants in search of the truth.
+
+The working group successfully implemented the database in PostgreSQL and GeoPackage formats and created forms for the classes (layers), including the rules for data integrity approved by requirements for the geospatial database, which were set up in QGIS. The symbol library with the rule-based symbology for each class (layers) was set up in the .qgs project file. Layer style settings for each layer were tested and saved into the geospatial database of the SPT.
+
+.. figure:: https://github.com/oleksaboiko/first/blob/683bd4095c8210e51a60684fcf9758a211828d5c/images/ukraine_spt1.png
```
Comment (DelazJ) [inline source/site/about/case_studies/ukrainian_spt]: Can you load this (and the other) image file in the [images folder](https://github.com/oleksaboiko/QGIS-Website/tree/patch-3/source/site/about/case_studies/images) and use that reference instead? See example in other rst file. It ensures that if you delete your branch (which is already the case), we do not lose the image in the website.

**Review by DelazJ — COMMENTED** [review]
Feedback for a successful build (sorry, I did not read the case study): you need to also reference this file in the [index.rst file](https://github.com/oleksaboiko/QGIS-Website/blob/patch-3/source/site/about/case_studies/index.rst) to have it built along with others.

**Comment by anitagraser** [comment]
Thank you, @oleksaboiko. Please share the images again. The links above appear broken.

**Comment by oleksaboiko** [comment]
Looks normal)

**Comment by oleksaboiko** [comment]
I`m sorry. It looks like I accidentally closed this pull request. 
Just a newbie on GitHub(

**Comment by DelazJ** [comment]
One last thing @oleksaboiko, your text file should be of .rst format. Can you turn that file into edit and add .rst to its name?

**Comment by oleksaboiko** [comment]
done)

**Pull request merged**